flatpak/mame.ini

```diff
@@ -0,0 +1,33 @@
+#
+# CORE SEARCH PATH OPTIONS
+#
+homepath                  $HOME/.mame
+rompath                   /app/share/mame/roms
+hashpath                  /app/share/mame/hash
+samplepath                /app/share/mame/samples
+artpath                   /app/share/mame/artwork
+ctrlrpath                 /app/share/mame/ctrlr
+inipath                   $HOME/.mame;/app/share/mame/ini
+fontpath                  /app/bin/
+cheatpath                 /app/share/mame/cheat
+crosshairpath             /app/share/mame/crosshair
+pluginspath               /app/share/mame/plugins
+languagepath              /app/share/mame/language
+swpath                    /app/share/mame/software
+
+#
+# CORE OUTPUT DIRECTORY OPTIONS
+#
+cfg_directory             $HOME/.mame/cfg
+nvram_directory           $HOME/.mame/nvram
+input_directory           $HOME/.mame/inp
+state_directory           $HOME/.mame/sta
+snapshot_directory        $HOME/.mame/snap
+diff_directory            $HOME/.mame/diff
+comment_directory         $HOME/.mame/comments
+share_directory           $HOME/.mame/share
+
+#
+# OSD VIDEO OPTIONS
+#
+video                     opengl
```

Comment:
Not sure I like this forcing to opengl, but I also legit don't have a strong opinion about what are the "unopinionated best" options for a *nix sandbox app at current time, roughly around:
- use X11 as videodriver backend (as it's currently doing) and deal with the terrible handling it has for Qt `-debug` on *nix? Or force bgfx, add Wayland (`USE_WAYLAND=1` as compile option below), and force above to use the imgui debugger given it's inside the same canvas so you don't have to deal with cross WM shenanigans?
- [Pipewire sounddriver vs. alsamixer](https://github.com/mamedev/mame/tree/sound), TBD